Multiple Reactions icon align with text

rt4914 opened the pull request:
<!--
     For Work In Progress Pull Requests, please use the Draft PR feature,
     see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.

     For a timely review/response, please avoid force-pushing additional
     commits if your PR already received reviews or comments.

     Before submitting a Pull Request, please ensure you've done the following:
     - 📖 Read the Forem Contributing Guide: https://developers.forem.com/contributing-guide/forem#create-a-pull-request
     - 📖 Read the Forem Code of Conduct: https://github.com/forem/forem/blob/main/CODE_OF_CONDUCT.md
     - 👷‍♀️ Create small PRs. In most cases this will be possible.
     - ✅ Provide tests for your changes.
     - 📝 Use descriptive commit messages.
     - 📗 Update any related documentation and include any relevant screenshots.

     NOTE: Pull Requests from forked repositories will need to be reviewed by
     a Forem Team member before any CI builds will run. Once your PR is approved
     with a `/ci` reply to the PR, it will be allowed to run subsequent builds without
     manual approval.
-->

## What type of PR is this? (check all applicable)

- [x] Refactor

## Description

Element alignment on #index. Example: https://forem.team/t/allhands (both regular and dark mode). The alignment looks off because of off the icons are aligned with the text. 

the multi-reaction count is not a clickable button as a comment for the article card.

![Feb-08-2023 09-39-11](https://user-images.githubusercontent.com/84626054/217431421-2145878f-3c76-4fff-82ea-0db8f901681b.gif)


## Related Tickets & Documents

<!--
For pull requests that relate or close an issue, please include them
below.  We like to follow [Github's guidance on linking issues to pull requests](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).

For example having the text: "closes #1234" would connect the current pull
request to issue 1234.  And when we merge the pull request, Github will
automatically close the issue.
-->

- Related Issue #19064

## QA Instructions, Screenshots, Recordings

| Before  | After |
| ------------- | ------------- |
| <img width="376" alt="Screenshot 2023-02-07 at 10 30 42 PM" src="https://user-images.githubusercontent.com/9396084/217334682-ce54b505-08ff-4500-8e49-060fca722bbe.png"> | <img width="348" alt="Screenshot 2023-02-07 at 10 30 13 PM" src="https://user-images.githubusercontent.com/9396084/217334672-bb8599ee-f90e-4a23-95e0-db8547f14424.png"> |
| ![Feb-08-2023 09-39-11](https://user-images.githubusercontent.com/84626054/217431421-2145878f-3c76-4fff-82ea-0db8f901681b.gif) | <img width="385" alt="Screenshot 2023-02-08 at 10 52 18 AM" src="https://user-images.githubusercontent.com/9396084/217442112-cc45866f-fe3f-48c4-a98d-771e4eb418ca.png"> |

On click on the `reactions + text` button in above image, it opens the article.

https://user-images.githubusercontent.com/9396084/217442343-06eb97de-d65a-44d9-b34d-3bdd0029facf.mov

### Added border to reactions icon 
<img width="335" alt="Screenshot 2023-02-08 at 12 07 52 PM (1)" src="https://user-images.githubusercontent.com/9396084/217456813-8e76085d-67e9-45ec-99de-2ca03f231315.png"> <img width="333" alt="Screenshot 2023-02-08 at 12 19 26 PM" src="https://user-images.githubusercontent.com/9396084/217456816-3f298281-0022-4911-8e84-d1f1fd1c0ea4.png">



### UI accessibility concerns?

No

## Added/updated tests?

- [x] No

## [Forem core team only] How will this change be communicated?

- [x] This change does not need to be communicated

anujbhavsar96 commented:
Overall, alignment stuff and clickable element looks perfect🚀

---

@rt4914 adding these two small tweaks here as we discussed

**1. Give multi-reaction icons border**
<img width="1061" alt="Screenshot 2023-02-08 at 11 21 13 AM" src="https://user-images.githubusercontent.com/84626054/217448133-5dacc134-fc0e-4a95-8936-7fd0095e939b.png">


**2. Change the icon order - Left most should be on top of the stack**
<img width="637" alt="Screenshot 2023-02-08 at 11 26 13 AM" src="https://user-images.githubusercontent.com/84626054/217448144-1a1198ee-58bf-421b-ba4e-b24d4a73cf26.png">

rt4914 commented:
> Overall, alignment stuff and clickable element looks perfect🚀
> 
> @rt4914 adding these two small tweaks here as we discussed
> 
> **1. Give multi-reaction icons border** <img alt="Screenshot 2023-02-08 at 11 21 13 AM" width="1061" src="https://user-images.githubusercontent.com/84626054/217448133-5dacc134-fc0e-4a95-8936-7fd0095e939b.png">
> 
> **2. Change the icon order - Left most should be on top of the stack** <img alt="Screenshot 2023-02-08 at 11 26 13 AM" width="637" src="https://user-images.githubusercontent.com/84626054/217448144-1a1198ee-58bf-421b-ba4e-b24d4a73cf26.png">

@anujbhavsar96 Update:

1. Added border and updated screenshots
2. Added as task to #19064

anujbhavsar96 reviewed (approved):
All three changes look good to me 🚀

Ridhwana commented on app/assets/stylesheets/views/article.scss:
```diff
@@ -460,22 +460,26 @@
   .crayons_icon_container {
     display: inline-block;
     background-color: var(--body-bg);
+    border: solid;
```
(suggestion): You could use the [border css shorthand property](https://developer.mozilla.org/en-US/docs/Web/CSS/border) here:

```
border: 2px solid var(--card-bg)
```

and remove `border: solid;`, ` border-color: var(--card-bg);`, `border-width: 2px;`

rt4914 replied:
Done. Thanks.

Ridhwana commented on app/views/articles/_single_story.html.erb:
```diff
@@ -121,18 +121,20 @@
-                                                                start: tag("span", { class: %w[hidden s:inline] }, true),
-                                                                end: "</span>".html_safe) %></span>
-              </div>
+              <a href="<%= story.path %>" class="crayons-btn crayons-btn--s crayons-btn--ghost crayons-btn--icon-left" data-reaction-count data-reactable-id="<%= story.id %>" aria-label="<%= t("views.articles.comments.aria_label", title: story.title) %>">
```
Nice catch on adding the link here.

rt4914 replied:
Thanks.

Ridhwana reviewed (approved):
Code looks good ✨Left one suggestion for the CSS.
